pages/index.js

```diff
@@ -78,61 +103,102 @@ export default function HomePage({ onAddEmotionEntry, handleToggleTooltip }) {
         input.
       </Tooltip>
+      <StyledFlexColumnWrapper>
+        <StyledTensionForm onSubmit={handleSubmit}>
+          <StyledTensionLabel htmlFor="tension-level">
+            On a scale from 0 to 100, how tense do you feel in this moment?
+          </StyledTensionLabel>
+          <StyledInput
+            aria-label="Adjust tension level between 0 and 100"
+            id="tension-level"
+            name="tensionLevel"
+            type="range"
+            value={tension}
+            max={100}
+            onChange={(event) => setTension(event.target.value)}
+          />
+          <StyledWrapper>
+            <StyledSpan>0</StyledSpan>
+            <StyledSpan>100</StyledSpan>
+          </StyledWrapper>
 
-      <StyledTensionForm onSubmit={handleSubmit}>
-        <StyledTensionLabel htmlFor="tension-level">
-          On a scale from 0 to 100, how tense do you feel in this moment?
-        </StyledTensionLabel>
-        <StyledInput
-          aria-label="Adjust tension level between 0 and 100"
-          id="tension-level"
-          name="tensionLevel"
-          type="range"
-          value={tension}
-          max={100}
-          onChange={(event) => setTension(event.target.value)}
-        />
-        <StyledWrapper>
-          <StyledSpan>0</StyledSpan>
-          <StyledSpan>100</StyledSpan>
-        </StyledWrapper>
-        {!isFormSubmitted && (
-          <>
-            <p>{tension}</p>
-            <StyledSaveButton type="submit">Save</StyledSaveButton>
-          </>
-        )}
+          {!isFormSubmitted && (
+            <>
+              <StyledTensionDisplay>{tension}</StyledTensionDisplay>
+              <SaveButton type="submit">Save</SaveButton>
+            </>
+          )}
 
-        {isFormSubmitted && (
-          <>
-            <StyledMessage>Your entry was successfully saved!</StyledMessage>
-            <StyledButtonWrapper>
-              <StyledBackButton
-                type="reset"
-                value={"Done"}
-                onClick={() => {
-                  setIsFormSubmitted(!isFormSubmitted);
-                  setTension("0");
-                }}
-              />
-              <StyledAddDetailsLink
-                $actionButton
-                href={{ pathname: "/create", query: { id: id } }}
-                forwardedAs={`/create`}
-              >
-                Add more details
-              </StyledAddDetailsLink>
-            </StyledButtonWrapper>
-          </>
+          {isFormSubmitted && (
+            <>
+              <StyledMessage>Your entry was successfully saved!</StyledMessage>
+              <StyledButtonWrapper>
+                <StyledBackButton
+                  type="reset"
+                  value={"Done"}
+                  onClick={() => {
+                    setIsFormSubmitted(!isFormSubmitted);
+                    setTension("0");
+                  }}
+                ></StyledBackButton>
```

Comment:
please use self closing tags if possible (whenever you don't pass children to a component)

Reply:
Done